app/src/main/java/com/kickstarter/libs/FirebaseAnalyticsClient.kt

```diff
@@ -28,4 +31,12 @@ open class FirebaseAnalyticsClient(
             }
         }
     }
+
+    override fun sendUserId(user: User) {
+        firebaseAnalytics?.let {
+            if (isEnabled()) {
+                firebaseAnalytics.setUserId(user.id().toString())
```

Comment:
Experiment here, will try to tie feature flag to specific userID, we are sending the userID only for admin user right now.